### ISSUE TITLE
 Support for multi-label and multi-class text classification using DistilBERT

### ISSUE THREAD

**Issue by emtropyml** [kaushaltrivedi/fast-bert#60]
How can I use DistilBERT for multi-label classification for building a fast and deploy-able model?

**Comment by kaushaltrivedi** [comment]
just set the model_type parameter to distilbert and the pretrained weights name as distilbert-base-uncased

**Comment by emtropyml** [comment]
Yeah, but what about DistilBertForMultiLabelSequenceClassification class ?

**Comment by mohammedayub44** [comment]
@kaushaltrivedi  How about for multilingual model. "DistilmBERT" (I want to run this for different languages)

Is the following config correct ?
modeltype='DistilmBERT' and 
tokenizer='distilbert-base-multilingual-cased' 

When I check `databunch.tokenizer`  it shows me `transformers.tokenization_distilbert.DistilBertTokenizer` 

Thanks !
